src/Autofac.Integration.AspNetCore.Multitenant/Autofac.Integration.AspNetCore.Multitenant.csproj

```diff
@@ -47,14 +47,16 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Autofac.Extensions.DependencyInjection" Version="7.2.0" />
+<!--    <PackageReference Include="Autofac.Extensions.DependencyInjection" Version="7.2.0" />-->
+    <PackageReference Include="Autofac.Extensions.DependencyInjection" Version="8.0.0" />
     <PackageReference Include="Autofac.Multitenant" Version="6.0.0" />
     <PackageReference Include="Microsoft.CodeAnalysis.Analyzers" Version="2.6.1">
       <PrivateAssets>All</PrivateAssets>
     </PackageReference>
     <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="2.6.1">
       <PrivateAssets>All</PrivateAssets>
     </PackageReference>
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="6.0.0" />
```

Comment:
There is a name clash because of `IServiceCollection` will see how to fix it.

Reply:
![image](https://user-images.githubusercontent.com/2729205/172413857-25d47c38-44d3-4966-a6f2-bf0ff1e3bc71.png)

The problem is that it has been part of `Microsoft.Extension.DependencyInjection.Abstractions` before `netcoreapp6.0`.

I am now conditionally including it so that lower versions retrieve the same package-reference.

Reply:
I tried that. I removed all conditional imports from the library project and I would still retrieve that error.

It is because of the framework-reference, which is different based on the TFM.

Basically this line --> `<FrameworkReference Include="Microsoft.AspNetCore.App" />`

Reply:
Can we... _not_ reference that? I mean, I guess I don't feel super strongly about the direct `<PackageReference>`, just thinking I don't want to get into a weird spot where there's some sort of transient dependency conflict. If you can't figure it out easily, I'm fine with leaving it.